src/lib/db/project-store.ts

```diff
@@ -134,7 +134,7 @@ class ProjectStore implements IProjectStore {
             memberCount.map((c) => [c.project, Number(c.count)]),
         );
         return projectsWithFeatureCount.map((r) => {
-            return { ...r, memberCount: memberMap.get(r.id) };
+            return { ...r, memberCount: memberMap.get(r.id) || 0 };
```

Comment:
What the hell is `r` ? Let's clean up these legacy one character identifiers while we are here.

Reply:
Usually that's a name clash with some other scope that triggers linter

Reply:
This one should be easy to clean up though. Didn't want to blow up the PR size but since you mentioned this I'm renaming those.